demisto_sdk/commands/setup_env/setup_environment.py

```diff
@@ -32,15 +33,16 @@
 )
 from demisto_sdk.commands.content_graph.objects.pack import Pack
 
+BACKUP_FILES_SUFFIX = ".demisto_sdk_backup"
```

Comment:
can you add it to gitignore in content?

Reply:
It's stored next to the original file within the `.idea` folder, so I believe it should be already ignored (since the whole folder is).